Expensify/App
# Dont toggle create menu if we already have a free policy

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/5392

### Tests / QA Steps

**Test that global create does not open when creating a policy view OldDot**
1. Log out of New Dot
1. Create a new account with a private domain via OldDot
1. Navigate to the group policies page and create a Free policy
1. Verify that you land in the Workspace Settings page in NewDot and the global create menu does not open

**Test that global create does not open when navigating to a free policy**
1. Log out of New Dot
1. Using the same account navigate to the group policies page and select the free policy
1. Verify that you land in the Workspace Settings page in NewDot and the global create menu does not open

**Test that logging into NewDot with a new account does open the menu**
1. Log out of New Dot
1. Create a new account with a private domain via NewDot and validate it
2. Verify that the create menu opens and has an option to create a workspace

**Test that logging into NewDot with a new account does not open the menu**
1. Log out of New Dot
1. Log in with the account you created just now and verify the create menu does not open

### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
